dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/metadata/metadata_import/MetadataImportBasedOnSchemasTest.java

```diff
@@ -104,7 +104,8 @@ public void postBasedOnSchema( String endpoint, String schema )
             "messageConversations",
             "users",
             "organisationUnitLevels",
-            "programRuleActions" ); // blacklisted because contains
+            "programRuleActions",
+            "programStages" ); // blacklisted because contains
```

Comment:
could we mark program as required in schemas? Then we should need to blacklist this

Reply:
ah, I see! LGTM then 👍